### ISSUE TITLE
Add deterministic random start value generator

### ISSUE THREAD

**Issue by maxschalz** [Nuclear-Verification-and-Disarmament/bicyclus#13]
This would be part of the `util` subpackage and could look something like this:
```python
def generate_start_value(sample_parameters, n=1):
    """Generate random start values. Ensure that the random generator has been deterministically seeded!"""
    def genstart():
        d = {}
        for (k, v) in sample_parameters.items():
            if type(v) is list:
                d[k] = np.random.random() * (v[1] - v[0]) + v[0]
            elif type(v) is dict:
                if v['type'] == 'Normal':
                    d[k] = (np.random.randn()*v['sigma']) + v['mu']
                elif v['type'] == 'TruncatedNormal':
                    d[k] = pm.TruncatedNormal.dist(**{p: pv for (p, pv) in v.items() if p != 'type'}).random()
                else:
                    assert False, f'Unknown parameter type {k} => {v}'
            else:
                assert False, f'Unknown parameter type {k} => {v}'
        return d

    if n == 1:
        return genstart()
    return [genstart() for i in range(0, n)]
```
Taken from Lewin Bormann's [Bayesian Cycle](https://git.rwth-aachen.de/nvd/fuel-cycle/bayesian-cycle). Note that `pm.XY.dist.random()` is PyMC3 and must be replaced by `pm.draw(pm.XY.dist(...))`, see [PyMC docs](https://www.pymc.io/projects/docs/en/latest/api/generated/pymc.draw.html?highlight=draw_values#pymc.draw).

If this is added, it is crucial that *reproducibility* is tested! AFAIK there's the possibility to pass a `seed` keyword to `pm.draw`. I should check if this is needed, or if I can fix the seeds before.

**Comment by maxschalz** [comment]
Conclusion: `random_seed` needs to be passed to both
1. `pm.draw(..., random_seed)`
2. `pm.sample(..., random_seed)`

This kwarg may also be a `numpy.RandomGenerator`: thus, a user can define one global RNG and pass it to the respective functions.

I have not yet been able to test this on different machines, but at least it works on one and the same machine.

**Comment by maxschalz** [comment]
Solved and closed, see 29477247cfe16e46a2030443317dbfd489237198 and #12 .
